src/xpk/commands/common.py

```diff
@@ -33,6 +33,7 @@ def set_cluster_command(args) -> int:
   command = (
       'gcloud container clusters get-credentials'
       f' {args.cluster} --region={zone_to_region(args.zone)}'
+      ' --dns-endpoint'
```

Comment:
I have the same question as @pawloch00 - what happens when a cluster is already created, without ' --enable-dns-access' . Does getting credentials with ' --dns-endpoint' still work?

Reply:
Thanks!

(Perhaps best to find out how we can determine whether dns access is enabled on a cluster or not. However, I don't think the PR needs to be blocked on it, since you've verified that the change is backward compatible.)